packages/node/src/local/client.ts

```diff
@@ -60,6 +68,26 @@ export class LocalEvaluationClient {
       this.config.flagConfigPollingIntervalMillis,
       this.config.debug,
     );
+    this.assignmentService = this.createAssignmentService();
+  }
+
+  private createAssignmentService(): AssignmentService | null {
+    if (this.config.assignmentConfiguration) {
+      const instance = amplitude.createInstance();
+      instance.init(
+        this.config.assignmentConfiguration.apiKey,
+        this.config.assignmentConfiguration,
+      );
+      const filterCapacity = this.config.assignmentConfiguration?.filterCapacity
+        ? this.config.assignmentConfiguration?.filterCapacity
+        : DEFAULT_FILTER_CAPACITY;
```

Comment:
Use object spreading to merge the actual input configuration with the default configuration if it exists. 

My logic here is hard to follow so let's pair on Monday to implement a more sane and javascript'y configuration logic.